wp-modules/pattern-data-handlers/pattern-data-handlers.php

```diff
@@ -123,9 +123,6 @@ function format_pattern_data( $pattern_data, $file ) {
 	include $file;
 	$pattern_data['content'] = ob_get_clean();
 
-	if ( ! $pattern_data['content'] ) {
-		return false;
-	}
```

Comment:
If a pattern has no content, this means `get_pattern_by_name()` will return `false`, even though the pattern file exists.

That means it won't get the meta, like Categories, and it'll overwrite the existing pattern file.

Reply:
Still, this could have side-effects in the rest of the plugin. It's a big change.

Reply:
Actually, `$pattern_data['content']` isn't falsy after publishing a pattern. 

For example, here's a newly published pattern with no content:

<img width="1020" alt="Screenshot 2023-05-15 at 12 20 23 PM" src="https://github.com/studiopress/pattern-manager/assets/4063887/77a116bc-a69c-4155-b429-3cc54925c136">

And here's what `get_pattern_by_name()` returns for it:

```sh
\W $wp shell
wp> PatternManager\PatternDataHandlers\get_pattern_by_name( 'my-new-pattern-15' )
=> array(7) {
  ["title"]=>
  string(17) "My New Pattern 15"
  ["slug"]=>
  string(17) "my-new-pattern-15"
  ["description"]=>
  string(0) ""
  ["viewportWidth"]=>
  int(1280)
  ["inserter"]=>
  bool(true)
  ["content"]=>
  string(1) "   <----------------------------- 'content' is not falsy
"
  ["name"]=>
  string(17) "my-new-pattern-15"
}

```

Reply:
So this isn't as big of a change as it might seem.

Reply:
So this `if` block doest nothing in that case.

It usually does nothing.

Reply:
Wow, it really is whack-a-mole 😵‍💫

Reply:
Haha, yeah